### PR TITLE
Laboratory/fix tests

### PR DESCRIPTION
**Pull request by GTOliver**
Fix some unit tests and TimeSpan behaviour

### CODE REVIEW

**Comment by Zephilinox** [comment]
could you prefix your commit messages with the project it changes? :)

File: Laboratory/Laboratory.csproj [hunk 1]
```diff
@@ -61,4 +61,21 @@
     <ProjectReference Include="..\Element.NET\Element.NET.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (Craig-J) [inline Laboratory/Laboratory.csproj:64]: These are all added by a glob pattern in line 22

File: Laboratory/Tests/L4-Standard-Library/ElementTimeSpan.cs [hunk 2]
```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Laboratory.Tests.L4.StandardLibrary
+{
+    internal class ElementTimeSpan : StandardLibraryFixture
```
Comment (Craig-J) [inline Laboratory/Tests/L4-Standard-Library/ElementTimeSpan.cs:7]: New tests should be written using ExpressionEvaluation or FunctionEvaluation to support both compile-time and runtime cases. See Numbers and Boolean tests in L3-Prelude for an example.

Comment (GTOliver) [reply Laboratory/Tests/L4-Standard-Library/ElementTimeSpan.cs:7]: Done :+1:

File: Laboratory/Laboratory.csproj (hunk 1 above)
Comment (GTOliver) [reply Laboratory/Laboratory.csproj]: Done 👍

**Review by Craig-J — APPROVED** [review]

**Pull request merged**